tests/lowering/eltwise/binary/test_div.py

```diff
@@ -78,3 +78,35 @@ def test_div_scalar_denom(device, input_shapes):
             assert node.meta["val"].size() == input_shapes[0]
     # Check inference result
     assert_with_pcc(result_before, result_after)
+
+
+@pytest.mark.parametrize(
+    "input_shapes",
+    [[(4, 4)], [(32, 32)]],
+)
+def test_div_scalar_numerator(device, input_shapes):
+    m = DivModule()
+    inputs = [torch.rand(shape, dtype=torch.bfloat16) for shape in input_shapes]
+    result_before = m.forward(5.0, *inputs)
```

Comment:
This test is for `scalar / tensor`.  There is already a test for `tensor / scalar` above.

Reply:
What about `scalar * tensor`?